doc/source/AOS_to_CCSDS.rst

```diff
@@ -9,8 +9,8 @@ config.yaml snippet:
 
 .. code-block:: none
 
-sle:
-  frame_output_port: 2568
+   - sle:
```

Comment:
Can you check if the hyphen is needed here?
Also, reviewing other YAML snippets in the file shows some sections misaligned, so please review those as well.  Thanks